schemas/Quiz.yaml

```diff
@@ -0,0 +1,17 @@
+title: Quiz
+type: object
+required:
+  - list of questions
+properties:
+  Questions:
+    description: Questions that makeup the Quiz
+    type: list
+    Question:
+      description: Phrase that demands a correct answer to
+      type: string
```

Comment:
the indentation all through this document is not correct.
please fix the indentation.